### PR TITLE
Initial Quick Start Guide added. Updated TOC using doctoc.

### PR DESCRIPTION
**Pull request by FeralBytes**
Please review my changes and feel free to critique as much as you want. Basically when I first started playing around with your project I read the Getting Started section and tried replicating them; but because the default behaviour of rapid script encapsulates your defined methods such as greet; everything I tried failed. But once I assigned a NameSpace to the window element I was able to start making the code work. So this Quick Start Guide will hopefully eliminate this misunderstanding.
I also had trouble performing AJAX at first but realized that it was due to browser protections; and was able to do it once the files were being served.
If you like this I would like to continue by editing the Getting Started section to reflect that it's code examples require you to execute the compiler using -b or --bare option so that your functions are accessible by say a onclick method. But I think advocating for NameSpaces is smarter so that people do not shoot themselves in the foot; that and NameSpaces are in the Zen of Python.
Also don't feel bad for nitpicking my changes I would rather they be of high quality thanks to a through review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +93,67 @@ From Git:
 If you're using OSX, you can probably use the same commands (let me know if that's not the case). If you're using Windows, you should be able to follow similar commands after installing node.js and git on your system.
 
 
+Quick Start Guide
+-----------------
+First lets creat a very minimal example in which we will have a simple html page that can be loaded in any modern browser and will display a button that will activate our Rapydscript JavaScript to alter the page.
```
Comment (atsepkov) [inline README.md]: "creat" typo

**Comment by atsepkov** [comment]
Thanks, haven't had the chance to read it in enough detail yet, just skimmed it so far. There is one typo I mentioned and, if you don't mind, I may tweak a couple sentences here and there to be more consistent with the writing style of surrounding paragraphs.

**Comment by FeralBytes** [comment]
OK I will fix the typo and update the pull once you give me the tweaks; consistency is definitely a good thing. 
Sorry I have not been very active; this time of year is very busy for me; and the building of my new website will not become important until closer to the end of the year; that is when I will be heavily using RapydScript.

**Comment by atsepkov** [comment]
Sorry for the delay, I've dropped the ball on this.

**Comment by FeralBytes** [comment]
No problem; we all get busy. I lately have been busy with work honestly; that is why I have not made more contributions. During the Winter I tend to have more time for programming and will be a more active contributor then.

**Pull request merged**